web/frontend/src/components/slam-2d-render.vue

```diff
@@ -7,42 +7,36 @@ import * as THREE from 'three';
 import { MapControls } from 'three/examples/jsm/controls/MapControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
-import { SVGLoader } from 'three/examples/jsm/loaders/SVGLoader';
-import { baseMarkerUrl } from '../lib/base-marker-url';
-import { destMarkerUrl } from '../lib/destination-marker-url';
-
-type SvgOffset = {
-  x: number,
-  y: number,
-  z: number
-}
+import DestMarker from '../lib/destination-marker.txt?raw';
+import BaseMarker from '../lib/base-marker.txt?raw';
+
+let pointsMaterial: THREE.PointsMaterial;
```

Comment:
`pointsMaterial` gets cached so that `size` can be tuned to the current camera zoom.